Classes/Public/Manager/Submanagers/OCTSubmanagerCalls.h

```diff
@@ -45,6 +45,16 @@
 - (OCTCall *)callToChat:(OCTChat *)chat enableAudio:(BOOL)enableAudio enableVideo:(BOOL)enableVideo error:(NSError **)error;
 
 /**
+ * Enable video calling for an active call.
+ * Use this when you started a call without video in the first place.
+ * @param enable YES to enable video, NO to stop video sending.
+ * @param call Call to enable video for.
+ * @param error Pointer to an error object.
+ * @return YES on success, otherwise NO.
+ */
+- (BOOL)enable:(BOOL)enable videoSendingForCall:(OCTCall *)call error:(NSError **)error;
```

Comment:
It would be better to rename this method. It is a little bit confusing now. I think that "first argument part" `enable:` should describe what the method is doing. While all other parts just describe other arguments.
```
- (BOOL)enableVideoSending:(BOOL)enable forCall:(OCTCall *)call error:(NSError **)error;
```